### DISCUSSION TITLE
Desktop release of Musify

### DISCUSSION THREAD

**Discussion by User1234-ui** [gokadzev/Musify#467] [General]
Hey there, I was just wondering will there be a desktop release of Musify? Would be dope!

**Comment by gokadzev** [comment]
> Hey there, I was just wondering will there be a desktop release of Musify? Would be dope!

Hello, it should be possible to do that, as I used to release several versions for Linux, and if I remember correctly, for Windows too. However, they didn’t have any users, and the project doesn’t provide enough benefits for me to justify adding more work to extend device compatibility
